cmd/tapcli/assets.go

```diff
@@ -178,16 +183,26 @@ func mintAsset(ctx *cli.Context) error {
 		}
 	}
 
+	assetType, err := parseAssetType(ctx)
+	if err != nil {
+		return err
+	}
+
+	amount := ctx.Uint64(assetSupplyName)
```

Comment:
What if we just set it to `1` for the user here? Can't be any other value anyway.

Reply:
That's what we currently do and is what lead to confusion: The user entered "5" and then didn't know why just one collectible was minted.

Reply:
Agree, this is one of the recurring reports - I think the only acceptable 'silent' behavior to have here would be to accept a mint request for a Collectible even if the supply flag is not set. But rejecting for amounts != 1 reduces surprise later on ("oh I guess I need to use an asset group to represent what i'd like to create")

Reply:
I changed the logic in the latest push in that we only silently update the amount to 1 if the `--supply` flag wasn't specified at all for collectible assets.